waltz-common/src/main/java/com/khartec/waltz/common/StringUtilities.java

```diff
@@ -113,4 +118,19 @@ public static int length(String str) {
                 ? 0
                 : str.length();
     }
+
+
+    public static String join(Collection<?> values, String separator) {
+        return values.stream()
+                .filter(v -> v != null)
+                .map(v -> v.toString())
+                .collect(Collectors.joining(separator));
+    }
+
+
+    public static <T> List<T> split(String value, String separator, Function<String, T> fromString) {
```

Comment:
Looks like plenty of ways for this go bang.  Perhaps `cnn` on `val` and `cnempty` on `sep`